ext/standard/tests/general_functions/proc_open_pipes3.phpt

```diff
@@ -28,17 +32,16 @@ echo "END\n";
 ?>
 --EXPECTF--
 Warning: proc_open(): pi is not a valid descriptor spec/mode in %s on line %d
-
-Warning: proc_open(): Descriptor item must be either an array or a File-Handle in %s on line %d
+Descriptor item must be either an array or a File-Handle
 array(4) {
   [3]=>
-  resource(%d) of type (Unknown)
+  resource(4) of type (Unknown)
```

Comment:
If you could revert them to using %d it would be better I think. Not exactly sure about resource number allocation.

Reply:
Yeah, sure. Somehow, it screwed them up.